example/src/main/java/com/xwray/groupie/example/MainActivity.kt

```diff
@@ -102,6 +92,16 @@ class MainActivity : AppCompatActivity() {
             })
         }
 
+        tracker = SelectionTracker.Builder<Long>(
```

Comment:
Adding this tracker to the whole recyclerview in the example breaks it. You should only add selection on the items that can actually be selected. From the looks of it, we should probably change `GroupieLookup` to only returns items if they can be selected